twitter/twitter.php

```diff
@@ -1279,7 +1279,7 @@ function twitter_fix_avatar($avatar)
 	$new_avatar = str_replace("_normal.", "_400x400.", $avatar);
 
 	$info = Images::getInfoFromURLCached($new_avatar);
-	if (!$info) {
+	if (empty($info)) {
```

Comment:
`empty()` is superfluous as empty array is natively cast as false in a condition.

Reply:
Once we have decided that in main repository's PR, I will change it here as well. :-)